cmake_configure.cmake

```diff
@@ -125,17 +125,42 @@ endif()
 
 include(CMakeParseArguments)
 
-if(CHECK_INCLUDES)
+if(CHECK_IWYU)
     find_package(IncludeWhatYouUse REQUIRED)
 endif()
 
+if(CHECK_CLANGTIDY)
+    find_program(CLANGTIDY_PROGRAM
+        clang-tidy
+    )
+    if(NOT CLANGTIDY_PROGRAM)
+        message(FATAL_ERROR "Cannot find clang-tidy")
+    endif()
+    set(CLANGTIDY_ARGS "${CLANGTIDY_PROGRAM}")
+    list(APPEND CLANGTIDY_ARGS
+        "-style=file"
```

Comment:
Shouldn't be here "-format-style=file"?

Reply:
The docs on http://clang.llvm.org/extra/clang-tidy/ show a `-format-style=file` option.
The version installed by fedora 26 does not have that option (version 4.0.1).
This maybe requires a `FindClangTidy.cmake` module if this option has changed.

Reply:
On arch I have to use "-format-style=file".

Reply:
I believe to have fixed it.